commit ab0e9e7b2b4acb28a44c998ee1928c672c365c0d
Author: Tariq Toukan <tariqt@mellanox.com>
Date:   Wed Feb 22 17:20:13 2017 +0200

    net/mlx5e: Do not reduce LRO WQE size when not using build_skb
    
    [ Upstream commit 4078e637c12f1e0a74293f1ec9563f42bff14a03 ]
    
    When rq_type is Striding RQ, no room of SKB_RESERVE is needed
    as SKB allocation is not done via build_skb.
    
    Fixes: e4b85508072b ("net/mlx5e: Slightly reduce hardware LRO size")
    Signed-off-by: Tariq Toukan <tariqt@mellanox.com>
    Signed-off-by: Saeed Mahameed <saeedm@mellanox.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1372_ab0e9e7b2b4acb28a44c998ee1928c672c365c0d.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1372_ab0e9e7b2b4acb28a44c998ee1928c672c365c0d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
